CommunityProject/Assets/_Game/Scripts/Runtime/Audio/ScriptableObjects/PlaylistItemSO.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace BoundfoxStudios.CommunityProject.Audio
+{
+	/// <summary>
+	/// <see cref="ScriptableObject"/> for holding a reference to an <see cref="AudioClip"/> and some Information about it.
+	/// </summary>
+	[CreateAssetMenu(menuName = Constants.MenuNames.Audio + "/Playlist Item")]
+    public class PlaylistItemSO : ScriptableObject
+    {
+		[SerializeField]
+		private AudioClip audioClip;
+
+		[SerializeField]
+		private string title;
+
+		[SerializeField]
+		private string interpreter;
+
+		[SerializeField]
+		private string url;
+
+		public AudioClip AudioClip => audioClip;
+		public string Title => title;
+		public string Interpreter => interpreter;
+		public string Url => url;
```

Comment:
Wie im Discord, hier nochmal der Vollständigkeithalber:

```suggestion
		public AudioClip AudioClip;
		public string Title;
		public string Interpreter;
		public string Url;
```

Streng genommen ist die Aufteilung natürlich sinnvoller, da man nur eine ReadOnly-API nach draußen anbietet, auf der anderen Seite wird ja auch das private Feld durch `SerializeField` quasi "öffentlich" und damit wieder beschreibbar. :)